### PR TITLE
Make the calendar great again

### PR DESCRIPTION
**Pull request by MagneDavidsen**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,13 +1,25 @@
...
   "version": "0.0.1",
   "description": "",
   "scripts": {
-    "preinstall": "cd src/frontend && npm install",
-    "install": "cd src/backend && npm install",
-    "start": "node src/backend/app.js",
-    "test": "cd src/frontend && npm test"
+    "start": "webpack-dev-server",
+    "dist": "webpack",
+    "watch": "webpack --watch"
```
Comment (ehamberg) [inline package.json]: Trailing whitespace

File: package.json [hunk 2]
```diff
@@ -1,13 +1,25 @@
...
-    "start": "node src/backend/app.js",
-    "test": "cd src/frontend && npm test"
+    "start": "webpack-dev-server",
+    "dist": "webpack",
+    "watch": "webpack --watch"    
   },
   "author": "",
-  "license": "Proprietary"
+  "license": "UNLICENSED",
+  "dependencies": {
```
Comment (ehamberg) [inline package.json]: Many/most of these should be dev dependencies.

**Comment by ehamberg** [comment]
The logic looks good.

A couple of (near-) show stoppers:

- You committed `src/dist/js/app.bundle.js` in `90c6789`
- Lots of dev dependencies included in build
- Quite a few uses of `var`, even in files where `const` (or `let`) is otherwise used
- Debugging printing left over

Almost all of these issues should be caught by something like eslint. Maybe you should base the eslint config on one of the other floq projects' so our es6 codebases are somewhat aligned. :)

**Comment by MagneDavidsen** [comment]
@ehamberg 

- Oops, that's the old dist-folder that I removed from .gitignore. My bad! Removing it now.
- Yes, I'll clean it up later
- Yes, I'll do a es2015-lint cleanup in one commit later
- I can't see that I added any debug printing?

**Comment by ehamberg** [comment]
Yep. Go-ahead for merging from me. The new stuff looks good. Let's do a clean-up after. :)

**Pull request merged**